src/components/ChatApp/VoiceRecognition.js

```diff
@@ -70,6 +70,10 @@ class VoiceRecognition extends Component {
     this.recognition.abort()
   }
 
+  onspeechstart = () => {
+    console.log('speechstart');
```

Comment:
can you explain the use case of the function? @zamhaq

Reply:
No use case. I added for some testing and forgot to remove it.

Reply:
Please remove this as well @zamhaq